test/functional/desired.js

```diff
@@ -45,7 +45,6 @@ const GENERIC_CAPS = node.deepFreeze({
     'appium:deviceName': DEVICE_NAME,
     'appium:automationName': 'XCUITest',
     'appium:launchWithIDB': LAUNCH_WITH_IDB,
-    'appium:noReset': true,
```

Comment:
why it is necessary to remove this cap?

Reply:
I wanted to avoid a situation where a test run makes the succeeding test runs fail. anyway, I reverted the capability and avoid the situation by https://github.com/appium/appium-xcuitest-driver/pull/1453/commits/7b917b8f23e3f78b7f1d437e12535c045163c731